src/main/java/com/thoughtfocus/springproject/controller/UpdateAgeController.java

```diff
@@ -20,7 +20,7 @@ String updateAge(@RequestBody UpdateAgeDto upagedto){
         }catch (Exception e){
             e.printStackTrace();
         }
```

Comment:
1. e.printStackTrace() needs to be removed as its sync call causes performance issue.
2. Format the code: use Ctrl+Shit+Alt+L to format in IntelliJ